specification/wse/src/main/scripts/org/kaazing/specification/wse/browsers/client.reconnect.downstream/request.rpt

```diff
@@ -16,16 +16,16 @@ read version "HTTP/1.1"
 read header "X-WebSocket-Version" "wseb-1.1"
 read header "Content-Type" "text/plain;charset=utf-8"
 
```

Comment:
This is fine, but I would also expect  /(?<upstream>http:\/\/localhost:8080\/path\/.+)\\n/ to work, right?